rpc_jobs/dep_update.yml

```diff
@@ -0,0 +1,40 @@
+- project:
+    name: "Dependency-Update-Jobs"
+    repo:
+      - rpc-openstack:
+          URL: "https://github.com/rcbops/rpc-openstack"
+          BRANCH: "newton"
```

Comment:
Why only newton?

Reply:
Mostly because I wanted to implement an example. If rpco want to update other branches or deps, they'll need to modify the jobs. OSA probably isn't the only dep that RPCO wants to track.